doc/sphinx/src/building.rst

```diff
@@ -12,42 +12,43 @@ propose an updated description in a pull request*
+|| REGRESSION\_GOLD\_STANDARD\_VER            || #                             || Int    || Version of current gold standard file used in regression tests. Default is set to latest version matching the source.                                       |
+|| REGRESSION\_GOLD\_STANDARD\_HASH           || SHA512=...                    || String || Hash value of gold standard file to be downloaded. Used to ensure that the download is not corrupted.                                                       |
+|| REGRESSION\_GOLD\_STANDARD\_SYNC           || ON                            || Option || Create `gold_standard` target to download gold standard files                                                                                               |
+|| CODE\_COVERAGE                             || OFF                           || Option || Builds with code coverage flags                                                                                                                             |
+|| PARTHENON\_LINT\_DEFAULT                   || OFF                           || Option || Lint the code as part of the default target (otherwise use the `lint` target)                                                                               |
+|| PARTHENON\_COPYRIGHT\_CHECK\_DEFAULT       || OFF                           || Option || Check copyright as part of the default target (otherwise use the `check-copyright` target)                                                                  |
+|| CMAKE\_INSTALL\_PREFIX                     || machine specific              || String || Optional path for library installation                                                                                                                      |
+|| Kokkos\_ROOT                               || unset                         || String || Path to a Kokkos source directory (containing CMakeLists.txt)                                                                                               |
+|| PARTHENON\_IMPORT\_KOKKOS                  || ON/OFF                        || Option || If ON, attempt to link to an external Kokkos library. If OFF, build Kokkos from source and package with Parthenon                                           |
+|| BUILD\_SHARED\_LIBS                        || OFF                           || Option || If installing Parthenon, whether to build as shared rather than static                                                                                      |
++---------------------------------------------+--------------------------------+---------+--------------------------------------------------------------------------------------------------------------------------------------------------------------+
```

Comment:
Weird that github can't (?) render this diff correctly since it should just be one added line

Reply:
I think it's all lines because the first column increased in number of characters.